### PR TITLE
[BUG] Account for half-pints when showing beer info

### PR DESCRIPTION
**Pull request by TomSeldon**
screen shot 2016-12-14 at 14 10 52

### CODE REVIEW

**Review by jenkoian — APPROVED** [review]
Nice, if only there was a half pint emoji...

**Comment by TomSeldon** [comment]
There's a (greenpint), but not much else...

File: src/taphouse.coffee [hunk 1]
```diff
@@ -34,9 +34,14 @@ module.exports = (robot) ->
...
 
                 drinks.forEach((drink) ->
-                    response.push(
-                      '(beer) ' + drink.name + ' [' + drink.brewery + '] - ' + drink.formattedPrice
-                    )
+                    output = '(beer) ' + drink.name + ' [' + drink.brewery + '] - '
+
+                    isHalfPint = drink.quantity == 'half'
+
+                    output = output + '½ ' if isHalfPint
```
Comment (rjwaring) [inline src/taphouse.coffee:41]: Yuck!

Comment (TomSeldon) [reply src/taphouse.coffee:41]: Thanks @rjwaring! 


...I had to write it like this. Originally wrote as a "normal" if statement and it complained. Looked at docs and saw this. It also complained when I used `===`, which isn't valid. It does automatically convert `==` to `===` though, so... (shrug)

**Pull request merged**